### DISCUSSION TITLE
Connection details?

### DISCUSSION THREAD

**Discussion by AviBueno** [prisma/prisma#4457] [Q&A] [answered]
Hi, is there a way to get the connection details (host/port/..) after a connection was established?

**Comment by ryands17** [comment, marked as answer]
Hey @AviBueno 👋 

Currently this is not supported by Prisma, so it would be great if you could open a feature request [here](https://github.com/prisma/prisma/issues/new?assignees=&labels=&template=feature_request.md&title=) and we will look into it :)

**Reply by AviBueno** [reply]
Thanks @ryands17, [done](https://github.com/prisma/prisma/issues/4461).

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
